Doc/whatsnew/3.10.rst

```diff
@@ -91,25 +91,40 @@ Optimizations
 =============
 
 
-Build and C API Changes
-=======================
-
-
-
 Deprecated
 ==========
 
 
-
 Removed
 =======
 
 
-
 Porting to Python 3.10
 ======================
 
 This section lists previously described changes and other bugfixes
 that may require changes to your code.
 
 
+
+Build Changes
+=============
+
+
+C API Changes
+=============
+
+New Features
+------------
+
+
+Porting to Python 3.10
+----------------------
+
+* Since :c:func:`Py_TYPE()` is changed to the inline static function,
+  ``Py_TYPE(obj) = new_type`` should be ported to use :c:func:`Py_SET_TYPE()`.
```

Comment:
```suggestion
  ``Py_TYPE(obj) = new_type`` must be replaced with ``Py_SET_TYPE(obj, new_type)``:
  see :c:func:`Py_SET_TYPE()` (available since Python 3.9).
```